src/components/plots/xyPlot.ts

```diff
@@ -158,69 +201,26 @@ module Plottable {
      * This call does not override auto domain adjustment behavior over visible points.
      */
     public showAllData() {
+      this._updateExtentsForAttr("x");
       this._xScale.autoDomain();
-      if (!this._autoAdjustYScaleDomain) {
-        this._yScale.autoDomain();
-      }
+      this._updateExtentsForAttr("y");
+      this._yScale.autoDomain();
+      return this;
     }
```

Comment:
It won't work with just calling `autoDomain()` on both scales, since we actually have to change the extents for this to work. Not sure if we want to keep it though, since the effect could be accomplished by shutting filtering off, calling autoDomain(), then turning the filtering back on.